### PR TITLE
Added Shared Select item for both Filter and header matching Modal

### PR DESCRIPTION
**Pull request by sadat66**
1. Created shared select field
2. Created Filter modal using shared select
3. Created csv mapping modal using shared select

### CODE REVIEW

File: src/components/layout/customer/components/expenses/newModal/FilterContent.tsx
Comment (ronybarua-ethical18) [inline src/components/layout/customer/components/expenses/newModal/FilterContent.tsx:1]: Need to make another shared component for similar pattern.

File: src/components/layout/customer/components/expenses/newModal/SelectField.tsx
Comment (ronybarua-ethical18) [inline src/components/layout/customer/components/expenses/newModal/SelectField.tsx:1]: The SelectField component should be placed on root directory of expense component

Comment (sadat66) [reply src/components/layout/customer/components/expenses/newModal/SelectField.tsx:1]: placed the components in the root folder of expense

File: src/components/layout/customer/components/expenses/newModal/FilterContent.tsx
Comment (sadat66) [reply src/components/layout/customer/components/expenses/newModal/FilterContent.tsx:1]: created another component

**Pull request merged**